src/shared/components/data-table/components/main/components/table-header/TableHeader.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useContext } from 'react';
+
+import { TableContext } from 'src/shared/components/data-table/DataTable';
+import TableHeaderCell from './components/table-header-cell/TableHeaderCell';
+
+import type { TableData } from 'src/shared/components/data-table/dataTableTypes';
+
+import styles from './TableHeader.scss';
+/*
+    - Should receive an array of header items along with it's configurations like tooltip, isSortable, isHighlighted
```

Comment:
`it's` -> `its` 🙂

Reply:
Changed it to `their` & `its` 👍

Reply:
The comment now reads: "Should receive an array of g items along with its configurations like tooltip, isSortable, isHighlighted"

- what's `g items`?
- I am still unclear about the configurations. Do you mean "configurations of the array" or "configurations of items"? If the latter, then it's `their configurations`